.devcontainer/Dockerfile

```diff
@@ -120,18 +120,18 @@ RUN echo "Install 3rd party dependencies" && \
         grpc-dev
 
 ##### Useful for logfile modification e.g. pruning all /magma/... prefix from GCC warning logs
-RUN GOBIN="/usr/bin/" go install github.com/ezekg/xo@0f7f076932dd
+RUN GOBIN="/usr/bin/" go install github.com/ezekg/xo@0f7f076932dd && \
+    rm --recursive --interactive=never /root/.cache/go-build
 
 ##### GRPC and it's dependencies
 RUN git clone --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc && \
-    cd grpc && \
-    mkdir -p cmake/build && \
-    cd cmake/build && \
+    mkdir -p grpc/cmake/build && \
+    cd grpc/cmake/build && \
     cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF -DBUILD_SHARED_LIBS=ON ../.. && \
     make -j"$(nproc)" && \
     make install && \
-    cd / && \
-    rm -rf grpc
+    cd ../../.. && \
+    rm --recursive --interactive=never grpc
```

Comment:
In contrast to `-f`, `--interactive=never` fails when the directory doesn't exist, so that we detect when this command isn't doing what we expect.

Reply:
Would one want to create an alias for it, if it is used multiple times throughout the script?